Development branch

meme-es opened the pull request:
This branch was used to develop the project, it includes the next files: index.html, style.css, image files and complementary files inside a well-defined folder tree.

me-skit committed:
Set-up linter files for remote repo and locally

In this commit, I setting-up the files to let Stickler works in the github repo,
the configuration includes the files: .stickler.yml and stylelint.config.js
Additionally I added a .gitignore file intended to ignore temporary files in
my local machine and to ignore the folder 'node_modules' which is intended
to let me know about linter errors in the local environment.

me-skit committed:
Add folders structure, basic files, and HTML markup

In this commit, I created the structure of the folders and I added  basic files:
index.thml, style.css, and images files. The HTML file contains the HTML markup
for the project

me-skit committed:
Add style to the project

This commit add content to the style.css file as well add font files and a couple
of images to finish the HTML appearance

me-skit committed:
Add a name class

In the last commit, a forgot to include some changes to the index.html file,
this change was just to add the 'nav-item' class for each <li> on the nav
inside the header

me-skit committed:
Add content to the README.md file

This commit adds content to the readme file to give info about the project, inside
the readme file there is a screenshot.png file embedded which was added too.

me-skit committed:
Modify the live demo surce in the readme file

I modifyed the url for the live demo, also I added the point indicated by the local
folder in some URL's of the style.css file

me-skit committed:
Fix URL issues

The URL of some images in the style.css file are not in the root of the project,
so those URL's were fixed.

me-skit committed:
Reedit the URL for the live demo

The last URL for the live demo doesn't work so, I used another page to show the
project

me-skit committed:
Reduce the nav-bar size

I reduced the width of the nav container.

euqueme commented on index.html:
```diff
@@ -0,0 +1,100 @@
+      </ul>
+    </nav>
+  </header>
+  <section class="main">
```
[OPTIONAL] change the `<section>` tag to be the `<main>` tag

meme-es replied:
Thank you, my team gave me the same suggestion on morning meeting, and it's already applied.

euqueme commented on assets/css/style.css:
```diff
@@ -0,0 +1,206 @@
+/* nav bar */
+
+.nav-container {
+  background: linear-gradient(to bottom, #8c8c8c 0%, #555 50%, #7c7c7c 52%, #a8a8a8 100%);
```
try with this gradient instead do you think it looks more like the original? :thinking: 

`background: linear-gradient(to bottom,gray 0%,#7c7c7c 50%,#666 51%,gray);`

meme-es replied:
Yeah! It looks more like the original, thank you again.

euqueme reviewed (approved):
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Congratulations @meme-es  :balloon:  :tada: :tada: 

### This project is just amazing :1st_place_medal: and I just have to approve it :clap: you followed the rules gracefully and there are really no major changes to make, I do have a couple of optional changes for you:

## Navbar Gradient
[OPTIONAL] I think the navbar is more on a grayscale maybe you can consider to change it a little to look even closer to the original.

## Building with Backgrounds

**Promo section**

[OPTIONAL] this project is all about building with backgrounds and gradients so consider to change all your `<img> `tags for `<div>` tags with the background image of the promos.

## Semantic HTML

[OPTIONAL] I Strongly encourage the use of the `<main>` tag for the dominant content of the `<body>` of a document

I have left comments in your code with my suggestions for the mentioned optional changes so feel free to try them in the inspector and keep them if you'd like to :arrow_double_down: 

### That's it! :raised_hands: 

### For more projects like this! :clinking_glasses: 

Happy Coding! :woman_technologist: 

PROJECT STATUS: APPROVED :green_apple:

me-skit committed:
Apply optional feature

I applied the optional feature which I consider are good soggestions:
- I used the <main> tag instead just a <div> for the dominant content
- I change the navbar gradient to be a bit more gray
